.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,86 @@
+name: Prepare Release
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        description: "Release version tag (e.g. v1.2.3)"
+        required: true
+      ref:
+        type: string
+        description: "Git ref from which to release"
+        required: true
+        default: "main"
+      do_publish_jars:
+        type: boolean
+        description: "Publish Java artifacts for staging"
+        required: true
+        default: "true"
+env:
+  NEXUS_SDK_REF: ${{ github.event.inputs.ref }}
+  NEXUS_SDK_TAG: ${{ github.event.inputs.tag }}
+
+jobs:
+  create_draft_release:
+    name: Create Github draft release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Audit gh version
+        run: gh --version
```

Comment:
Is this necessary?

Reply:
Not _necessary_ but it demonstrates the CLI version in use. This was taken from the Temporal Java SDK library at https://github.com/temporalio/sdk-java/blob/add6c4e8174b81f31ebc373142844bd9c89c4b9d/.github/workflows/prepare-release.yml#L36-L37 presumably because it was valuable information to someone at one time